HHS/Head-Start-TTADP
# WYSIWYG & Report Downloads Production deploy

## Description of change

### Big changes

* Add a What You See Is What You Get (WYSIWYG) editor to various Activity Report fields: https://ocio-jira.acf.hhs.gov/browse/TTAHUB-26
* Allow downloading of _all_ reports in the result set, based on the selected filters: https://ocio-jira.acf.hhs.gov/browse/TTAHUB-103

### Smaller changes

* Context menu for last row of activity reports table is now properly visible at all times: https://ocio-jira.acf.hhs.gov/browse/TTAHUB-117
* Remove "edit" button on report viewing page when the report can no longer be editable: https://ocio-jira.acf.hhs.gov/browse/TTAHUB-65


## How to test

1. Create a new report and add formatting to the context or TTA provided fields (WYSIWYG editor)
2. Filter reports by date and select "Reports" -> "Export Table Data"
3. Select "..." menu for the last row and verify buttons are not hidden
4. View an approved report and verify that the "edit" links are no longer visible


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
